### PR TITLE
replace pngs with vector drawable xmls

### LINKED ISSUE #414: Use vector drawables instead of PNGs
There are a lot of PNGs being used in the code. Replace them with equivalent xml drawable files

### Reference
https://sendeyo.com/onlineconverter/en/image-png/xml

### PR DESCRIPTION
**Pull request by lifeLessCoder**
## Description
Replaced some of the pngs with xmls.

Fixes #414 

## Type of change
Just put an x in the [] which are valid.
- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
The app has been run on an android device and tested

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

File: Android/app/src/main/java/io/github/project_travel_mate/HomeFragment.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/HomeFragment.java]: why this change?

File: Android/app/src/main/java/io/github/project_travel_mate/favourite/FavouriteCitiesAdapter.java [hunk 2]
```diff
@@ -11,8 +11,6 @@
 
 import com.squareup.picasso.Picasso;
 
-import org.json.JSONArray;
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/favourite/FavouriteCitiesAdapter.java]: All these refactoring change shouldnt be a part of this PR

**Comment by lifeLessCoder** [comment]
There are many unused pngs in the drawable folder. The issue #594 refers to this. I'll soon work on it
